dev-requirements.txt

```diff
@@ -0,0 +1,40 @@
+#### TESTING-RELATED PACKAGES ####
+
+# Checks style, syntax, and other useful errors.
+flake8==3.7.8
```

Comment:
Do we need to pin these?

Reply:
I don't know--but these are just copied from the old `requirements.txt`.  It'd be nice not to pin `black` and `flake8` (which shouldn't change much).  I'm going to merge this PR and unpin `black` and `flake8` in a follow up PR.

My guess is that we need to pin `mypy`.